### DISCUSSION TITLE
Snapshot Timed Out Help

### DISCUSSION THREAD

**Discussion by chevyman142000** [koush/scrypted#692] [Q&A]
Hello! I'm seeing the following error in my log after noticiing that I'm seeing Snapshot Timed Out in Homekit. I have 3 Wyze V3 cameras running their RTSP firmware and all 3 are having the same issue. Let me know what else I can provide to help troubleshoot. Thanks for the help!

[Back Yard Cam] Snapshot failed w [Error]: 
    at takePicture (/src/main.ts:284:27)
    at func (/src/main.ts:297:33)
    at P.snapshotDebouncer (/common/src/promise-utils.ts:85:21)
    at P.takePicture (/src/main.ts:294:37)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at RpcPeer.handleMessageInternal (/server/node_modules/@scrypted/server/src/rpc.ts:577:37)
[Back Yard Cam] creating error image with background Snapshot Unavailable true

![IMG_3027](https://user-images.githubusercontent.com/4703038/228702127-d749d0b5-b4c0-4f0e-800c-e80278849485.jpg)

**Comment by swav8tor** [comment]
I have 1 v2 and 2 pan, all working great.  Just purchased a new v3 cam and are running into same issue.  All 3 original are working fine, just the new one doesn’t give screenshot, video is working however.

![1AEB6D96-CA14-41CD-9A2B-CDB7A7F7CE9B](https://user-images.githubusercontent.com/17955875/231503284-fbe6d8d0-d672-48c1-aab6-3211219a51b0.jpeg)

**Reply by NPSconsultingLLC** [reply]
I too am having this issue with the V3 cameras.

Pi 4 (64 bit os)

**Reply by swav8tor** [reply]
I ended up returning the Wyze and got the TP link basic , they work great.
